ts/src/oxfun.ts

```diff
@@ -0,0 +1,2843 @@
+        return this.parseOrder (order);
+    }
+
+    async cancelAllOrders (symbol: Str = undefined, params = {}) {
+        /**
+         * @method
+         * @name oxfun#cancelAllOrders
+         * @description cancel all open orders
+         * @see https://docs.ox.fun/?json#delete-v3-orders-cancel-all
+         * @param {string} symbol unified market symbol, only orders in the market of this symbol are cancelled when symbol is not undefined
+         * @param {object} [params] extra parameters specific to the exchange API endpoint
+         * @returns {object} response from exchange
+         */
+        const request: Dict = {};
+        if (symbol !== undefined) {
+            const market = this.market (symbol);
+            request['marketCode'] = market['id'];
+        }
+        //
+        //     {
+        //         "success": true,
+        //         "data": { "notice": "Orders queued for cancelation" }
+        //     }
+        //
+        //     {
+        //         "success": true,
+        //         "data": { "notice": "No working orders found" }
+        //     }
+        //
+        return await this.privateDeleteV3OrdersCancelAll (this.extend (request, params));
+    }
+
+    async cancelOrders (ids:string[], symbol: Str = undefined, params = {}) {
+        /**
+         * @method
+         * @name oxfun#cancelOrders
+         * @description cancel multiple orders
+         * @see https://docs.ox.fun/?json#delete-v3-orders-cancel
+         * @param {string[]} ids order ids
+         * @param {string} [symbol] unified market symbol
+         * @param {object} [params] extra parameters specific to the exchange API endpoint
+         * @param {int} [params.timestamp] in milliseconds
+         * @param {int} [params.recvWindow] in milliseconds
+         * @param {string} [params.responseType] 'FULL' or 'ACK'
+         * @returns {object} an list of [order structures]{@link https://docs.ccxt.com/#/?id=order-structure}
+         */
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' cancelOrders() requires a symbol argument');
+        }
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const marketId = market['id'];
+        const request: Dict = {
+            'timestamp': this.milliseconds (),
+            'responseType': 'FULL',
+        };
+        const orders = [];
+        for (let i = 0; i < ids.length; i++) {
+            const order = {
+                'marketCode': marketId,
+                'orderId': ids[i],
+            };
+            orders.push (order);
+        }
+        request['orders'] = orders;
+        const response = await this.privateDeleteV3OrdersCancel (this.extend (request, params));
+        const data = this.safeList (response, 'data', []);
+        return this.parseOrders (data, market);
+    }
+
+    parseOrder (order, market: Market = undefined): Order {
+        //
```

Comment:
can we add a sample here?

Reply:
Yeah. We will add (there's a lot of samples)